### DISCUSSION TITLE
Run Jolokia in Intelliji got error

### DISCUSSION THREAD

**Discussion by NamVuHai** [rhuss/jolokia#659] [Q&A] [answered]
Hi community,
I'm tried to custom Jolokia and run it in Intelliji, but seem didn't work.
Application give error: Error opening zip file or JAR manifest missing: C:\Users\Desktop\jolokia-1.7.2\jolokia-1.7.2\agent\jvm\target\classes
But if running file jar after complie, it still work.
Anyone can help me out

**Comment by NamVuHai** [comment, marked as answer]
i found the solution 
https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/4828

**Comment by grgrzybek** [comment]
Next time please create an issue with `Question` label please ;)
